strongbox-commons/src/main/java/org/carlspring/strongbox/io/SbByteRangeInputStream.java

```diff
@@ -0,0 +1,98 @@
+package org.carlspring.strongbox.io;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.security.NoSuchAlgorithmException;
+import java.util.List;
+
+import org.carlspring.commons.http.range.ByteRange;
+import org.carlspring.commons.io.AbstractByteRangeInputStream;
+import org.carlspring.commons.io.reloading.ReloadableInputStreamHandler;
+
+public class SbByteRangeInputStream extends AbstractByteRangeInputStream
```

Comment:
Why did this functionality need to be moved?
I don't like the name of the class - it should be `ByteRangeInputStream`. I believe there was a class called like that in `org.carlspring.commons:commons-io`.